templates/task-sc.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/version: {{ .Chart.Version }}
 {{- if .Values.annotations }}
   annotations:
-  {{- .Values.annotations | toYaml | nindent 4 }}
+{{- .Values.annotations | toYaml | nindent 4 }}
```

Comment:
I recommend keeping the indentation block in place. TLDR; there are two entities going on, the actual YAML file, and the template logic thus deserves its own indentation level.